en/services/gimbal_v2.md

```diff
@@ -255,6 +255,19 @@ This signals that the gimbal manager also is also acting as the gimbal device.
 
 Also see [how to address non-MAVLink gimbal devices](#non_mavlink_gimbal_device_addressing).
 
+#### How to interpret `GIMBAL_DEVICE_ATTITUDE_STATUS` yaw gimbal angle
+
+The [GIMBAL_DEVICE_ATTITUDE_STATUS.flags](#GIMBAL_DEVICE_ATTITUDE_STATUS) field must report the frame used for reported yaw values as **either**:
+
+- `GIMBAL_DEVICE_FLAGS_YAW_IN_VEHICLE_FRAME`: Yaw is relative to vehicle.
+- `GIMBAL_DEVICE_FLAGS_YAW_IN_EARTH_FRAME`: Yaw is relative to north.
+
+Older devices may use the deprecated `GIMBAL_DEVICE_FLAGS_YAW_LOCK` to indicate that the yaw is relative to north, and un-set to indicate that yaw is relative to the vehicle.
+This should only be checked if the two flags above are not set!
```

Comment:
```suggestion
For older devices, if neither of the flags above are set, the yaw frame has be inferred from the `GIMBAL_DEVICE_FLAGS_YAW_LOCK`. If it is set, the yaw is relative to North, otherwise to forward of the vehicle.

Reply:
Thanks @julianoes - minor subedit and merging.